learn/cookbooks/docker.md

```diff
@@ -0,0 +1,135 @@
+# Using Meilisearch with Docker
+
+A common way of installing Meilisearch is using Docker. In this guide you will learn what Docker is, how to install it in your local machine, and how to use Meilisearch with it.
+
+## Docker
+
+Docker is a tool that packages an application's code and dependencies into something called a container. Using Docker allows developers to ensure their software runs reliably across a wide range of machines and environments.
+
+Docker containers are distributed in images. Docker images are packages storing not only the container, but also instructions on how Docker should instantiate that container.
+
+## Install Docker
```

Comment:
We should also share a small note for those developers who don't have GUI, for instance if they are using a Cloud instance of Linux. So adding instructions for using Docker CLI

Reply:
Hi Shivay! I wanted to do that, but Docker seems to have gone out of its way to remove Docker CLI mentions from anywhere remotely useful. Since it's not our job to explain things Docker doesn't, I decided to write the guide from the assumption that people using Docker on remote machines know what they're doing.